macos/Sources/Features/Terminal/TerminalWindow.swift

```diff
@@ -560,6 +549,11 @@ class TerminalWindow: NSWindow {
 
         // Hide the window drag handle.
         windowDragHandle?.isHidden = true
+
+        // Reenable the main toolbar title
+        if let toolbar = toolbar as? TerminalToolbar {
+            toolbar.titleIsHidden = false
+        }
     }
```

Comment:
Perhaps this function should be renamed to something like `resetCustomTabBarViews` now that it's responsible for both hiding some elements and unhiding others? (The current name is `hideCustomTabBarViews`, see 3 lines up.)